iothub/service/src/Twin/Models/TwinCollection.cs

```diff
@@ -160,7 +160,7 @@ public TwinMetadata GetMetadata()
         /// Gets the last updated time for this property.
         /// </summary>
         /// <returns>Date-time instance representing the last updated time for this property.</returns>
-        /// <exception cref="System.NullReferenceException">Thrown when the metadata object is null.
+        /// <exception cref="NullReferenceException">Thrown when the metadata object is null.
```

Comment:
This was a bug we couldn't fix, but we should fix now. Not in this PR, but we should fix it so we don't return an NRE.

Reply:
An SDK should never throw a NullReferenceException. That is a bug.